tests/test_config.py

```diff
@@ -51,6 +52,7 @@ def contain_horsepower(df):
 
 def test_default_actions_registered(global_var):
     df = pytest.car_df
+    lux.config.set_executor_type("Pandas")
     df._repr_html_()
```

Comment:
why do we need line 55? If it is set to `SQL` those methods should handle the issue? Maybe the expected will change, but I'm not sure.

Reply:
In the test_compiler script we set the global executor type to be SQLExecutor in the last test so we need to change the executor type back to Pandas here, otherwise none of the data gathering will work with the dataframe and a warning will be thrown.

Looking back at test_config, the `set_SQL_connection()` call in `register_new_action()` acctually seems redundant. For this script we just need to set the executor to be a PandasExecutor in the first test function.